src/org/labkey/test/components/experiment/LineageGraph.java

```diff
@@ -0,0 +1,105 @@
+package org.labkey.test.components.experiment;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class LineageGraph extends WebDriverComponent<LineageGraph.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+
+    public LineageGraph(WebElement element, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+    }
+
+    /**
+     * finds the list of edges/nodes to the one currently focused, by name
+     * @param listTitle
+     * @return
+     */
+    public NodeDetails getDetails(String listTitle)
+    {
+        return elementCache().summaryList(listTitle);
```

Comment:
This doesn't return a list, it returns a single NodeDetails item. The NodeDetails may contain a list of NodeDetailItems .